src/utils/types.ts

```diff
@@ -0,0 +1,64 @@
+import { Nullable } from "tsdef"
+
+export type Ticket = {
+  id: number
+  adults: Nullable<number>
+  children: Nullable<number>
+  count: number
+  description: string
+  date_created: string
+  date_updated: Nullable<string>
+  expirationTimestamp: Nullable<string>
+  expirationTimestampSane: string
+  has_pets: boolean
+  location: Nullable<any>
+  location_to: Nullable<any>
+  need_tag_id: NeedTag[] | []
+  need_type: Nullable<string>
+  organization_id: Nullable<Organization>
+  phone: string
+  phone_public: boolean
+  ticket_status: string
+  user_created: Nullable<string>
+  user_updated: Nullable<string>
+  visits: number
+  what: string
+  where: string
+  where_from_tag: Nullable<string>
+  where_to: Nullable<string>
+  where_to_tag: Nullable<string>
+  who: string
+}
```

Comment:
**suggestion:**
I would strongly avoid generating such Complete types. 

Because we operate on an STI model, we should use interface segregation to make life simpler for and end developer

Reply:
fixed